reana_client/cli/workflow.py

```diff
@@ -210,14 +216,22 @@ def workflow_create(ctx, file, name, skip_validation, access_token):
          'Overrides value of REANA_WORKON.')
 @add_access_token_options
 @click.option(
-    '-p', '--parameter',
+    '-p', '--parameters',
```

Comment:
The plural is not user friendly here:

```console
$  reana-client start --parameters year_min=1800 --parameters region=Oceania
```

One would rather expect:

```console
$  reana-client start --parameter year_min=1800 --parameter region=Ocenia
```

because each item is a parameter, not a list in itself.

P.S. Ditto for `--options` vs `--option`.

Reply:
What I meant is to change the name in the code, like [here](https://github.com/pallets/click/issues/725#issuecomment-298497992), and keep singular for the user option (each --parameter call is only one parameter, but the Python variable will contain all the specified parameters).